mods/d2k/languages/lua/en.ftl

```diff
@@ -0,0 +1,147 @@
+## campaign.lua
+objective-failed = Objective failed
+objective-completed = Objective completed
+
+primary = Primary
+secondary = Secondary
+
+new-primary-objective = New primary objective
+new-secondary-objective = New secondary objectives
+
+mentat = Mentat
+
+## atreides-01ab, atreides03ab, harkonnen-01a, ordos-05
+harvested-resources = Harvested resources: { $harvested }/{ $goal }
```

Comment:
This uses parameters now, so the scripts have to use parameters as well. (E.g. atreides01b)